common/config/defaults.go

```diff
@@ -15,6 +15,8 @@ const (
 	DefaultEnvironmentType         = models.EnvironmentTypeDynamic
 	DefaultEnvironmentOS           = "linux"
 	DefaultServiceScale            = 1
+	DefaultRetryTimeout            = time.Second * 60
```

Comment:
this probably isn't required; the different api calls should each use retry and sleep parameters suited for that particular call; e.g. the calls for determining if the security group is deleted should be much slower than something like checking the loadbalancer is deleted.